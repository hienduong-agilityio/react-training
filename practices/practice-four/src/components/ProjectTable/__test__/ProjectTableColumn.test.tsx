// Libraries
import { render } from '@testing-library/react';

// Components
import ProjectTableColumn from '@/components/ProjectTable/ProjectTableColumn';

describe('ProjectTableColumn Component', () => {
  it('should render and match snapshot', () => {
    const { container } = render(<ProjectTableColumn />);
    expect(container).toMatchSnapshot();
  });
});
