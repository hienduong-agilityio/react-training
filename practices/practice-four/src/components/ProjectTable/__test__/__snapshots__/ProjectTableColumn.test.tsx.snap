// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`ProjectTableColumn Component should render and match snapshot 1`] = `
<div>
  <thead
    class="bg-gray-50"
  >
    <tr
      class="h-9 uppercase"
    >
      <th
        class="pl-3 font-medium text-gray-500"
      >
        Id
      </th>
      <th
        class="font-medium text-gray-500"
      >
        Project Name
      </th>
      <th
        class="text-center font-medium text-gray-500"
      >
        PM
      </th>
      <th
        class="font-medium text-gray-500"
      >
        Status
      </th>
      <th
        class="font-medium text-gray-500"
      >
        Last update
      </th>
      <th
        class="text-center font-medium text-gray-500"
      >
        Resources
      </th>
      <th
        class="font-medium text-gray-500"
      >
        Project timeline
      </th>
      <th
        class="font-medium text-gray-500"
      >
        Estimation
      </th>
    </tr>
  </thead>
</div>
`;
